src/api/create-repo.js

```diff
@@ -0,0 +1,21 @@
+const { StatusCodes } = require('http-status-codes');
+
+const generateUniqueName = (username) => {
+ return `${username}-photohub-${Math.floor(Math.random() * 10000)}`;
```

Comment:
Why not use the current Date? "humphd-photohub-06-nov-2022" or something?

Reply:
That's another option. I wanted to keep the name short. Also, what if the multiple repos is generated simultaneously by accident or through automation? With the current implementation, it may crash the server with an error.

Reply:
@humphd I was not sure if you wanted to allow the creation of multiple repos for the same user. I added a unique identifier to enable the creation of multiple repos to store photos. We can check if the user already had a repo generated using my-photohub and not allow them to create a new repo. Another option I could think of is to delete the repo generated using my-phothub and replace it with a brand-new one each time. The other option is to let the user specify the repo name.

Reply:
That's another option. I wanted to keep the name short. Also, what if the multiple repos is generated simultaneously by accident or through automation? With the current implementation, it may crash the server with an error.

Reply:
@humphd We can either prompt the user for the `repo` name, as an alternative. It seems like the API complains when a repo already exists on the user's GitHub account. We can do one of the following:
- Ask the user to provide a repo name
- Use the unique identifier only if the user already has one repo generated using `my-photohub`
- Delete the existing repo and generate a fresh one (this one may require additional permission during PAT token generation)